samples/highcharts/data-tools/mathmodifier-formula/demo.js

```diff
@@ -0,0 +1,163 @@
+        'Disposed meals': [1, 2, 4, 8, 16, 32, '-', '=AVERAGE(C1:C6)'],
+        'Total meals': [
+            '=SUM(B1:C1)', '=SUM(B2:C2)', '=SUM(B3:C3)', '=SUM(B4:C4)',
+            '=SUM(B5:C5)', '=SUM(B6:C6)', '-',
+            '=SUM(B8:C8)'
+        ]
+    }
+});
+
+
+// Setup & Render Table
+
+table.setModifier(new MathModifier());
+
+renderTable(container1, table, true);
+renderTable(container2, table.modified);
+
+
+// Add Change Listener
+
+function onChange(e) {
+    console.log(e);
+    if (e.key === 'Enter') {
+        const td = e.target;
+        const column = Array
+            .from(td.parentNode.childNodes)
+            .indexOf(td);
+        const row = Array
+            .from(td.parentNode.parentNode.childNodes)
+            .indexOf(td.parentNode);
+        const value = parseFloat(td.innerText);
+
+        e.preventDefault();
+        td.blur();
+
+        table.setCell(
+            table.getColumnNames()[column],
+            row,
+            (isNaN(value) ? td.innerText : value)
+        );
+
+        renderTable(container2, table.modified);
+
+        cursor.emitCursor(table, {
+            column: Array
+                .from(td.parentNode.childNodes)
+                .indexOf(td),
+            row: Array
+                .from(td.parentNode.parentNode.childNodes)
+                .indexOf(td.parentNode),
+            state: 'table.mouseover',
+            type: 'position'
+        }, e);
+    }
+}
+
+container1.querySelector('tbody').addEventListener('keydown', onChange);
+
+
+// Synchronize MouseOver
+
+function synchronizeMouseOver(e) {
+    const td = e.target;
+
+    if (td.tagName !== 'TD') {
+        return;
+    }
+
+    cursor.emitCursor(table, {
+        column: Array
+            .from(td.parentNode.childNodes)
+            .indexOf(td),
+        row: Array
+            .from(td.parentNode.parentNode.childNodes)
+            .indexOf(td.parentNode),
+        state: 'table.mouseover',
+        type: 'position'
+    }, e);
+}
+
+container1.addEventListener('mouseover', synchronizeMouseOver);
+container2.addEventListener('mouseover', synchronizeMouseOver);
+
+function synchronizeCursor(e) {
+    const cursor = e.cursor;
+
+    synchronizeHighlight(container1, cursor);
+    synchronizeHighlight(container2, cursor);
+}
+
+cursor.addListener(table.id, 'table.mouseover', synchronizeCursor);
+
+function synchronizeHighlight(container, cursor) {
+    const tbody = container.querySelector('tbody');
+    const trHighlight = tbody.querySelectorAll('tr')[cursor.row];
+    const tdHighlight = trHighlight.querySelectorAll('td')[cursor.column];
+
+    for (const tr of tbody.querySelectorAll('tr')) {
+        // We search for the correct HTML row
+        if (tr === trHighlight) {
+            tr.classList.add('highlight');
+        } else {
+            tr.classList.remove('highlight');
+        }
+        for (const td of tr.querySelectorAll('td')) {
```

Comment:
The same situation like in `tr`

Reply:
This is a different situation as it selects only `td`s from the current `tr`.